recipes-bsp/u-boot/u-boot-boundary-fw-utils/aarch64/fw_env.config

```diff
@@ -1 +1 @@
-/dev/mmcblk0boot0	0x1fe000	0x2000		0x1000
+/dev/mmcblk0boot0	0x3fe000	0x2000		0x1000
```

Comment:
Fix a kinda misleading. What was happening before? Was it working?

Changing offset is something with huge side effects for end-users as it may break their previously working devices.

Reply:
The old offset was not working. With our latest uboot, the new offset is needed or else fw utils will not work.